test/e2e/subscription_e2e_test.go

```diff
@@ -75,6 +75,12 @@ var _ = Describe("Subscription", func() {
 		csv, err := fetchCSV(crc, subscription.Status.CurrentCSV, testNamespace, buildCSVConditionChecker(v1alpha1.CSVPhaseSucceeded))
 		require.NoError(GinkgoT(), err)
 
+		// Ensure that the OperatorCondition was created for the CSV.
```

Comment:
I think we should also be interested in ensuring the existence of the automagically generated RBAC for OperatorConditions.

Reply:
Agreed - I'm working on that test locally.

Reply:
Addressed in the latest version of the PR.